src/runtime/subscription-linking-flow-test.js

```diff
@@ -80,21 +91,71 @@ describes.realWin('SubscriptionLinkingFlow', (env) => {
       });
       dialogManagerMock.verify();
     });
+
+    it('throws an error if publisherProvidedId is missing', async () => {
+      const request = {...REQUEST, publisherProvidedId: undefined};
+      await expect(
+        subscriptionLinkingFlow.start(request)
+      ).to.eventually.be.rejectedWith('publisherProvidedId');
+    });
   });
 
-  it('throws an error if publisherProvidedId is missing', async () => {
-    const request = {...REQUEST, publisherProvidedId: undefined};
-    await expect(
-      subscriptionLinkingFlow.start(request)
-    ).to.eventually.be.rejectedWith('publisherProvidedId');
+  describe('startMultipleLinks', () => {
+    it('opens /linksaveiframe in an ActivityIframeView', async () => {
+      let activityIframeView;
+      let hidden;
+      let dialogConfig;
+      dialogManagerMock
+        .expects('openView')
+        .once()
+        .callsFake((viewArg, hiddenArg, dialogConfigArg) => {
+          activityIframeView = viewArg;
+          hidden = hiddenArg;
+          dialogConfig = dialogConfigArg;
+          return Promise.resolve();
+        });
+
+      subscriptionLinkingFlow.startMultipleLinks(MULTI_REQUEST);
+
+      const url = new URL(activityIframeView.src_);
```

Comment:
don't love that we depend on the implementation details of activityIframeView. Can we mock it and use the public interface like other tests?

Reply:
What do you think about this instead?

Reply:
I was able to implement that, but I had to inject new promises so it could wait for things to finish rendering before running the checks.

Reply:
LGTM. Thanks!